package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "atob": "^2.1.2",
     "body-parser": "^1.19.0",
+    "clightningjs": "0.0.2",
```

Comment:
May need to fork this library before adding it to our dependencies. Vary of adding freshly developed libraries, due to security reasons.

Reply:
it is from a vary active c-lightning collaborator

https://github.com/ElementsProject/lightning/commits?author=darosior

Reply:
Will merge in that case. Thanks.